blocks/columns/columns.css

```diff
@@ -1,11 +1,15 @@
 .columns > div {
     display: flex;
     flex-direction: column;
-    }
+}
+
+.columns > div > div > *:first-child {
+    margin-top: 0;
+}
```

Comment:
Is this too greedy to remove top margin from all first children of a column cell?

Reply:
Yeah, actually there needs to be more space between the text and the images now that I see the Figma design Keith posted yesterday.
![image](https://user-images.githubusercontent.com/3883795/228232557-b4117282-9dc6-4e31-b378-de583fe2c498.png)